tests/wiseagents/yaml_env/test_yaml_env.py

```diff
@@ -0,0 +1,231 @@
+import yaml
+
+# This appears to be unused but actually does something!
+import wiseagents.yaml_env
```

Comment:
Just curious, can `from wiseagents.yaml_env import setup` be used here?

Reply:
Yes, it can - I preferred to keep it namespaced though.

Thinking about it, that is because 'setup()' is quite generic. Perhaps I should change it to `setup_yaml_for_env_vars()` or something like that?

Reply:
Yeah, that sounds like a better name.

Reply:
@fjuma Updated